README.md

```diff
@@ -36,6 +36,42 @@ This will fetch the latest stable version of CLOVER and install it into your cur
 
 To download the CLOVER source code directly from Github, simply click the green `Code` button near the top of this page, and select `Download ZIP`. Once downloaded, unpack the zip file into a directory of your choice. You will now be able to run CLOVER from a terminal in this directory. Use the `cd` command to change the directory of your terminal to the extracted folder in order to run CLOVER.
 
+#### Downloading Git LFS files
```

Comment:
Overall I think this process looks fine, especially at this stage. Does the implementation of this in the long term depend on the limitations of the Git LFS? Or somewhere else to host these files?

Reply:
The git lfs currently doesn't work, as per #40 , where Ilaria has hit an error whilst trying to sync the files as we've exceeded are storage quote.. I think we probably want to fix this then before merging in these changes :smile:

Reply:
@phil-sandwell , this process has now been changed to use Zenodo, see updated REAMDE.MD file :smile: